CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
 namespace CompanyEmployees.Core.Domain.Entities
```

Comment:
The new `Salary` property is missing data annotations such as `[Required]` or `[Column(TypeName = "decimal(18,2)")]` which are typically used for monetary values in EF Core to ensure precision and validation.

```suggestion
[Column(TypeName = "decimal(18,2)")]
public decimal Salary { get; set; }
```